deploy/vagrant/provision/roles/ufs_glusterfs/files/config_tachyon.sh

```diff
@@ -1,8 +1,3 @@
 #!/bin/bash
 
-# last should be master
-namenode=`tail -n1 /tachyon/conf/workers`
-
-# config tachyon env
-/bin/cp -f /tachyon/conf/tachyon-glusterfs-env.sh.template /tachyon/conf/tachyon-env.sh
-sed -i "s/export TACHYON_MASTER_ADDRESS=localhost/export TACHYON_MASTER_ADDRESS=${namenode}/g" /tachyon/conf/tachyon-env.sh
+# Nothing to do
```

Comment:
Yes, `roles/ufs_{ufs_type}/files/config_tachyon.sh is always included by `roles/tachyon/tasks/config.yml`.